Source/HelixToolkit.Maths/BoundingBoxHelper.cs

```diff
@@ -3,33 +3,33 @@
     /// <summary>
     /// 
     /// </summary>
-    public static class BoundingBoxExtensions
+    public static class BoundingBoxHelper
     {
         /// <summary>
         /// Get bounding box from list of points
         /// </summary>
-        /// <param name="points"></param>
+        /// <param name="points">The points.</param>
         /// <returns></returns>
-        public static BoundingBox GetBoundingBox(this IList<Vector3> points)
+        public static BoundingBox FromPoints(IList<Vector3>? points)
         {
             if (points == null || points.Count == 0)
             {
                 return new BoundingBox();
             }
-            Vector3 min = new(float.MaxValue);
-            Vector3 max = new(float.MinValue);
+            var min = new Vector3(float.MaxValue);
```

Comment:
Why do we want to change this back? I changed it because of the visual studio suggestion from latest C# standard.